.travis-build.sh

```diff
@@ -6,6 +6,40 @@ set -e
 
 export SHELLOPTS
 
+# Optional argument $1 is one of:
+#   cfi-tests, downstream
+# If it is omitted, this script does everything.
+export GROUP=$1
+if [[ "${GROUP}" == "" ]]; then
+  export GROUP=all
+fi
+
+SLUGOWNER=${TRAVIS_REPO_SLUG%/*}
+if [[ "$SLUGOWNER" == "" ]]; then
+  SLUGOWNER=opprop
+fi
+
+if [[ "${GROUP}" != "cfi-tests" && "${GROUP}" != "downstream" && "${GROUP}" != "all" ]]; then
+  echo "Bad argument '${GROUP}'; should be omitted or one of: cfi-tests, downstream, all."
+  exit 1
+fi
+
 . ./.travis-build-without-test.sh
 
-ant -f tests.xml run-tests
+# Test CF Inference
+if [[ "${GROUP}" == "cfi-tests" || "${GROUP}" == "all" ]]; then
+    ant -f tests.xml run-tests
+fi
+
+# Downstream tests
+if [[ "${GROUP}" == "downstream" || "${GROUP}" == "all" ]]; then
```

Comment:
Do condition check on `SLUGOWNER`, only perform down stream test on ontology in opprop.